src/components/loader-spinner/loader-spinner.test.tsx

```diff
@@ -104,13 +101,12 @@ describe("when custom props are passed", () => {
   });
 
   it("should override the visually hidden label text when the 'spinnerLabel' prop is passed a custom string value", () => {
-    mockUseMediaQuery.mockReturnValueOnce(false);
     render(<LoaderSpinner spinnerLabel="bar" showSpinnerLabel={false} />);
 
     const status = screen.getByRole("status");
     const hiddenLabelElement = within(status).getByText("bar");
 
-    expect(hiddenLabelElement).toBeInTheDocument();
+    expect(hiddenLabelElement).not.toBeVisible();
```

Comment:
comment: I think it's worth keeping the assertion as before but adding the new one etc. That way we're asserting that the label is there but not visible etc
```
expect(hiddenLabelElement).not.toBeVisible();
expect(hiddenLabelElement).toBeInTheDocument();

Reply:
as mentioned above it looks like the hidden label will be visible to RTL so lets go with instead
```
const visibleLabelElement = screen.queryByTestId("visible-label"); expect(visibleLabelElement).not.toBeInTheDocument();